elements/bulbs-video/components/revealed.js

```diff
@@ -107,7 +105,8 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, playerOptions) {
-    new VideoPlayer(element, playerOptions); // eslint-disable-line no-new
+    let player = new VideoPlayer(element, playerOptions);
+    player.player.play();
```

Comment:
This only fires when the <bulbs-video> switches to the `revealed` state. By default that only happens when you click the dormant player. If the element is `<bulbs-video autplay>` it is automatically kicked into the `revealed` state.